src/ashish_Vyas/Assignment_12/ExampleReturn1.java

```diff
@@ -0,0 +1,62 @@
+/*Assignment 12 : 5th April 2021
+Design to test static, non static, return type concepts.
+Sample input1 : 
+10,20,30
+Sample output1 : 
+Candidate is Eligible
+Candidate obtained B grade. 
+ * 
+ */
+
+package ashish_Vyas.Assignment_12;
+
+public class ExampleReturn1 {
+
+	//1. Method sum() - non static method, accept 3 parameters of type double & return answer
+	double sum(double x, double y, double z){
+		return z+x+y ;
+	}
+	
+	//2. Method average()- non static method, accept 3 parameters of type double & return answer
+	double average(double x, double y, double z){
+		
+		return (z+x+y)/2 ;
```

Comment:
Here, this average mean, you need to devide number's sum with total number and here total number is 3 not 2